src/components/BaseModalSelectItem.vue

```diff
@@ -23,7 +23,7 @@ defineProps<{
           />
           <BasePill>{{ tag }}</BasePill>
         </div>
-        <span class="text-skin-text" v-text="description" />
+        <span class="text-skin-text break-all" v-text="description" />
```

Comment:
I added this because addresses cause formatting issues, and so we need to word-break them.

let me know if this change is not okay, and I'll create a variant of this component with the word breaks and use that here instead.